tests/v2/validation/publicapi/clusterroletemplatebindings/crtb_create_steve_test.go

```diff
@@ -0,0 +1,307 @@
+package clusterroletemplatebindings
+
+import (
+	"encoding/json"
+	"testing"
+
+	v1 "github.com/rancher/rancher/pkg/apis/provisioning.cattle.io/v1"
+	management "github.com/rancher/rancher/tests/framework/clients/rancher/generated/management/v3"
+	namegen "github.com/rancher/rancher/tests/framework/pkg/namegenerator"
+
+	"github.com/rancher/rancher/tests/framework/clients/rancher"
+	"github.com/rancher/rancher/tests/framework/extensions/clusters"
+	"github.com/rancher/rancher/tests/framework/extensions/users"
+	"github.com/rancher/rancher/tests/framework/pkg/session"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+)
+
+type CRTBGenTestSuite struct {
+	suite.Suite
+	client            *rancher.Client
+	crtbName          string
+	roleTemplateName  string
+	userPrincipalName string
+	session           *session.Session
+	cluster           *management.Cluster
+	clusterName       string
+	remoteClusterName string
+	clusterID         string
+	remoteClusterID   string
+	clusterV1         *v1.Cluster
+	jsonData          []byte
+}
+
+const (
+	crtbJSONFilePath        = "resources/createCRTBTemplate.json"
+	crtbApiEndPoint         = "management.cattle.io.clusterroletemplatebinding"
+	defaultRoleTemplateName = "cluster-member"
+)
+
+func (crtb *CRTBGenTestSuite) TearDownSuite() {
+	crtb.session.Cleanup()
+}
+
+func (crtb *CRTBGenTestSuite) SetupSuite() {
+	testSession := session.NewSession()
+	crtb.session = testSession
+
+	client, err := rancher.NewClient("", testSession)
+	require.NoError(crtb.T(), err)
+
+	crtb.client = client
+	crtb.clusterName = client.RancherConfig.ClusterName
+	require.NotEmptyf(crtb.T(), crtb.clusterName, "Cluster name to install should be set")
+
+	crtb.clusterID, err = clusters.GetClusterIDByName(crtb.client, crtb.clusterName)
+	require.NoError(crtb.T(), err, "Error getting cluster ID")
+	crtb.cluster, err = crtb.client.Management.Cluster.ByID(crtb.clusterID)
```

Comment:
Is this really necessary? I looked through the rest of the script and I don't see these being used anywhere.